orttraining/orttraining/training_ops/cuda/triton/triton_op.cc

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifdef ENABLE_TRITON
```

Comment:
nit: this is not needed because we only include this file if ENABLE_TRITON is defined and this happens in providers.cmake.

Reply:
Tried that but it caused trouble for AMD build, guess it's related to hipify, need more change in provider.cmake to make it work. So currently still follow the way as torch_interop did, we can fix this in the future for both torch_interop and triton (maybe also aten).